b2b_ecommerce/factories.py

```diff
@@ -24,7 +24,7 @@ class B2BOrderFactory(DjangoModelFactory):
     total_price = factory.LazyAttribute(lambda obj: obj.per_item_price * obj.num_seats)
     product_version = factory.SubFactory(ProductVersionFactory)
     coupon_payment_version = factory.SubFactory(CouponPaymentVersionFactory)
-    contract_number = str(fuzzy.FuzzyInteger(low=100, high=1000))
+    contract_number = fuzzy.FuzzyInteger(low=100, high=1000)
```

Comment:
Since this is a string field in the model, maybe use something like this instead:
```python
contract_number = fuzzy.FuzzyChoice(["", "abc123", "123456"])
```